locations/spiders/Amazons.py

```diff
@@ -0,0 +1,130 @@
+                            time.append(str(int(time[1][0])+12)+":00")
+                        if days:
+                            if len(days) == 2:
+                                one, two = days_dict[days[0]], days_dict[days[1]]
+                                if two > one:
+                                    for i in range(one,two+1):
+                                        openhours.append({"Day":days_rev_dict[i],"open":time[2],"close":time[3]})
+                                else:
+                                    for j in range(two,one+1):
+                                        openhours.append({"Day":days_rev_dict[i],"open":time[2],"close":time[3]})
+                            else:
+                                openhours.append({"Day":days[0],"open":time[2],"close":time[3]})
+                    except Exception:
+                        continue
+                    if not (j in [':',': '] and j in ':\n') and len(j) < 20:
+                        ok = j.strip(':').strip('\n')
+                        if ok == 'To come':
+                            continue
+                        if len(ok) > 12:
+                            if ok[0] == '(':
+                                item["phone"] = ok
+                            elif ok not in '\n' and ok not in ' ':
+                                item["Timings"] = ok
+                if '\n' in add:
+                    add = add.replace("\n"," ")
+                div = add.split(',')
+                some = add.split(' ')
+                for i in range(len(some)):
+                    if ',' in some[i] and len(some[i-1])<4 and i >1:
+                        item["extras"] = some[i-1] + " "+some[i]
+                if some[0][0].isdigit():
+                    item["housenumber"] = some[0]
+                    item["street"] = some[1] + " "+some[2]
+                if len(div) == 2:
+                    try:
+                        item["city"]= div[0].split(' ')[-1]
+                        item["state"] = div[1].split(' ')[1]
+                        item["postcode"] = div[1].split(' ')[2]
+                    except Exception as ex:
+                        item["city"] = div[0].split(' ')[-2]+" "+div[0].split(' ')[-1]
+                        item["postcode"] = div[1]
+                        item["state"] = div[0].split(' ')[-1]
+                item["addr_full"] = add
+                if locations:
+                    item["lat"] = locations[0]
+                    item["lon"] = locations[1]
+                else:
+                    item["lat"] = None
+                    item["lon"] = None
+                try:
+                    item['phone']
+                except Exception:
+                    item['phone'] = None
+                if openhours:
+                    item["opening_hours"] = openhours
+                item.pop("Timings",None)
+                break
+        if item and len(item) > 2:
+            yield GeojsonPointItem(**item)
```

Comment:
You aren't setting a `ref` value on GeojsonPointItem.

Reply:
Ohh ok done